src/compas/data/encoders.py

```diff
@@ -97,9 +97,9 @@ def object_hook(self, o):
             raise DecoderError("The data type of the object should be in the following format: '{}/{}'".format(o.__class__.__module__, o.__class__.__name__))
 
         except ImportError:
-            raise DecoderError("The module of the data type can't be found.")
+            raise DecoderError("The module of the data type can't be found: {}.".format(o['dtype']))
```

Comment:
is 'dtype' guaranteed as a key of `o`?  would `o.get('dtype')` be safer?

Reply:
it is because otherwise there would have been an earlier return